.travis.yml

```diff
@@ -9,11 +9,13 @@ notifications:
 env:
   - MIX_ENV=test
 before_script:
-  - mix compile 
+  - mix deps.get
+  - mix compile --warnings-as-errors
   - MIX_ENV=dev travis_wait 20 mix dialyzer --plt
 
 script:
-  - mix deps.get
+  - mix format --check-formatted
```

Comment:
This is already done else where and was taken out for a reason.

Reply:
re: https://github.com/pixelunion/elixir-shopifyapi/blob/master/.github/main.workflow

Reply:
Yup. So it is. I will remove